package.json

```diff
@@ -16,80 +16,90 @@
     "version": "npm run build && git add -A lib",
     "postversion": "git push && git push --tags"
   },
+  "engines": {
+    "node": ">=6.x.x",
+    "npm": ">=5.4.x"
+  },
   "peerDependencies": {
-    "react": "15.4.2",
-    "react-dom": "15.4.2"
+    "react": "15.4.x",
```

Comment:
Should we only have react and react-dom as peer dependencies. Other dependencies should be bundled with the grid.

Reply:
ok, done.